registry/get-help.md

```diff
@@ -1,3 +1 @@
-[the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
-
--	**Need a free, secure hosted registry?** Try [Docker Hub](https://www.docker.com/pricing).
```

Comment:
This was added over in #1798 in conversation with @j0hnst0n - I guess now that Distribution is CNCF, this is a bit more complicated. :grimacing:

(Apologies if a conversation around this already happened and the removal was intentional/pre-discussed!)

Reply:
There was a discussion about all removing references to Docker (Hub) from all docs now that distribution has been donated to CNCF, yeah. Hence this change.

Reply:
https://github.com/docker-library/docs/pull/2113/commits/ffb8ea0a91cd22a64143cd75cb151de3c8a0a1db